tools/src/main/scala/scala/scalanative/build/LLVM.scala

```diff
@@ -146,6 +154,12 @@ private[scalanative] object LLVM {
       case lto      => Seq(s"-flto=${lto.name}")
     }
 
+  private def asan(config: Config): Seq[String] =
```

Comment:
Why do we need to enable address sanitiser?  It should not be a part of 32bit support PR

Reply:
Had to use ASAN so we could get some bugs fixed for 32-bit support, so ended up adding support for it since we need to do things like disable unwinding when it's enabled.